rasa/test.py

```diff
@@ -27,6 +32,35 @@ def test_core_models_in_directory(model_directory: Text, stories: Text, output:
     plot_core_results(output, number_of_stories)
 
 
+def _get_sanitized_model_directory(model_directory: Text) -> Text:
+    """Adjusts the '--model' argument of 'rasa test core' when called with '--evaluate-model-directory'.
+
+    If a file is passed as 'model_directory' to 'test_core_models_in_directory' instead of a directory, returns the
```

Comment:
I think this is even a bit too much of information considering that it's such a short function. Also it focus on the 'how' instead of why. How about 

```suggestion
    By default rasa uses the latest model for the `--model` parameter. However, for `--evaluate-model-directory` we need a directory. This function checks if the passed parameter is a model or an individual model file.
```